pocket_coffea/workflows/genweights.py

```diff
@@ -0,0 +1,53 @@
+import awkward as ak
+
+import copy
+
+from .base import BaseProcessorABC
+from ..utils.configurator import Configurator
+
+class genWeightsProcessor(BaseProcessorABC):
```

Comment:
No need to derive it from the BaseProcessorABC, we are not using any facility of it

Reply:
Or do you want to use some function of it? OK

Reply:
The idea is to use the same dict structure and same metadata parameters as defined in the `BaseProcessorABC`.